src/commands/utility/mostplayed.js

```diff
@@ -0,0 +1,44 @@
+const { Command, SwitchbladeEmbed, MiscUtils } = require('../../')
+
+module.exports = class MostPlayed extends Command {
+  constructor (client) {
+    super({
+      name: 'mostplayed',
+      aliases: ['topgames'],
+      category: 'utility',
+      requirements: { guildOnly: true },
+      parameters: [{
+        type: 'guild',
+        full: true,
+        required: false
+      }]
```

Comment:
Honestly, I don't think this should be allowed. (sharing data from a different guild)

Reply:
will fix